etc/scripts/build.sh

```diff
@@ -29,25 +29,21 @@ trap on_error ERR
 
 # Path to this script
 if [ -h "${0}" ] ; then
-  readonly SCRIPT_PATH="$(readlink "${0}")"
+    readonly SCRIPT_PATH="$(readlink "${0}")"
 else
-  readonly SCRIPT_PATH="${0}"
+    readonly SCRIPT_PATH="${0}"
 fi
 
 # Path to the root of the workspace
 readonly WS_DIR=$(cd $(dirname -- "${SCRIPT_PATH}") ; cd ../.. ; pwd -P)
 
 source ${WS_DIR}/etc/scripts/pipeline-env.sh
 
-if [ "${WERCKER}" = "true" -o "${GITLAB}" = "true" ] ; then
-  apt-get update && apt-get -y install graphviz
-fi
-
-inject_credentials
+mvn ${MAVEN_ARGS} --version
 
-mvn -f ${WS_DIR}/pom.xml \
+mvn ${MAVEN_ARGS} -f ${WS_DIR}/pom.xml \
     clean install -e \
-    -B \
-    -Pexamples,integrations,archetypes,spotbugs,javadoc,docs,sources,tck,tests,pipeline
+    -Dmaven.test.failure.ignore=true \
```

Comment:
(Confirming that you want to remove `-B` and it wasn't a typo.)

Reply:
Yeah, this is part of ${MAVEN_ARGS} now.

Reply:
This is the same as what we have on master.